src/components/scenes/Loans/ManageCollateralScene.tsx

```diff
@@ -69,11 +89,13 @@ export const ManageCollateralScene = <T extends keyof ParamList>(props: Props<T>
   const { borrowEngine, borrowPlugin } = loanAccount
   const { currencyWallet: borrowEngineWallet } = loanAccount.borrowEngine
   const { fiatCurrencyCode: isoFiatCurrencyCode, currencyInfo: borrowEngineCurrencyInfo } = borrowEngineWallet
+  const actionOpStrings = ACTIONOP_TYPE_STRING_MAP[actionOpType]
   const collaterals = useWatch(borrowEngine, 'collaterals')
   const debts = useWatch(borrowEngine, 'debts')
   const borrowEnginePluginId = borrowEngineCurrencyInfo.pluginId
   const borrowPluginInfo = borrowPlugin.borrowInfo
   const borrowPluginId = borrowPluginInfo.borrowPluginId
+  const isShowAprChange = actionOpType === 'loan-repay' || actionOpType === 'loan-borrow'
```

Comment:
Name suggestion: `isAprChangeShown`. I think if the verb is always going to be "is", thne we have to figure out a noun form for any verbs like "show".

Reply:
Yeah, your suggestion is a nicer **sounding** name, but I would like to caution trying to mold this concept into 'proper English.'

'Proper English' is rife with pitfalls in that the English language itself is just plain weird and has a lot of exceptions and arbitrary rules for tense, plurality, etc. French is similar in this regard. As such, IMO:

**Consistency and hierarchy > Nice sounding English**

Less overhead in naming things too if there's a simple system in place without having to wordsmith.

I know also you are a fan of imperatives, so, `isShowApr` might be the best here in that sense

Reply:
It's all arguable I suppose. With that said, the claim `Consistency and hierarchy > Nice sounding English` is conjecture without some predicate to back it the inequality. If you don't want to change it, that's A-OK.